lib/utils/typeGuards.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ * @param {import('postcss').Node} node
+ * @returns {node is import('postcss').Root}
+ */
+exports.isRoot = function isRoot(node) {
```

Comment:
What is this file for?

Also, I think it should be:

```suggestion
module.exports.isRoot = function isRoot(node) {
```

Reply:
for this things https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types